cloud/cluster/memory/cluster.go

```diff
@@ -0,0 +1,138 @@
+package memory
```

Comment:
Is this really an in memory fake like inMemoryQueue or inMemorySagaLog?

shouldn't it just be part of cluster like dynamicCluster was, we don't plan on replacing this with a durable version.  The idea was that this class is reused and however you do service discovery updates just sends updates to this class/object